roles/ansible/tower/manage-projects/templates/project.j2

```diff
@@ -7,7 +7,11 @@
     "scm_branch": "{{ project.scm_branch | default('master') }}",
     "scm_clean": {{ project.scm_clean | default(false) }},
     "scm_delete_on_update": {{ project.scm_delete_on_update | default(false) }},
+    {% if (credential_id is defined) and (credential_id != "") %}
```

Comment:
Can we add `credential_id|trim != ""` ? I think we've bumped into issues before where something gets through and the empty space causes this to get evaluated